test/dbservice_web/controllers/group_controller_test.exs

```diff
@@ -53,21 +87,20 @@ defmodule DbserviceWeb.GroupControllerTest do
     setup [:create_group]
 
     test "renders group when data is valid", %{conn: conn, group: %Group{id: id} = group} do
-      conn = put(conn, Routes.group_path(conn, :update, group), group: @update_attrs)
-      assert %{"id" => ^id} = json_response(conn, 200)["data"]
+      conn = put(conn, Routes.group_path(conn, :update, group), @update_attrs)
+      assert %{"id" => ^id} = json_response(conn, 200)
```

Comment:
what does `^id` mean here?

Reply:
We use the pin operator ^ when you want to pattern match against a variable’s existing value rather than rebinding the variable. For example:
iex> x = 1
1
iex> [^x, 2, 3] = [1, 2, 3]